metadata.yaml

```diff
@@ -14,23 +14,32 @@ description: |-
   with all solid runoff converted to liquid runoff with no heat transfer.
   Spin up starts from a nominal year of 1 Jan 1900.
 notes: |-
-  Source code: https://github.com/COSIMA/access-om2
-  License: Attribution-NonCommercial-NoDerivatives 4.0 International (http://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
-  Conditions of use:
-  We request that users of this or other ACCESS-OM2 model code or output data:
+  COSIMA request that users of this or other ACCESS-OM2 model code or output data:
   (a) consider citing Kiss et al. (2020) [http://doi.org/10.5194/gmd-13-401-2020]
   (b) include an acknowledgement such as the following:
   "The authors thank the Consortium for Ocean-Sea Ice Modelling in Australia (COSIMA; http://www.cosima.org.au)
   for making the ACCESS-OM2 suite of models available at https://github.com/COSIMA/access-om2.
   Model runs were undertaken with the assistance of resources from the National Computational Infrastructure (NCI),
   which is supported by the Australian Government."
-  (c) let us know of any publications which use these models or data so we can add them to our list:
+  (c) let COSIMA know of any publications which use these models or data so they can add them to their list:
   https://scholar.google.com/citations?hl=en&user=inVqu_4AAAAJ
 keywords:
-  - tenth
-  - cosima
-  - access-om2-01
-  - jra55-do-v1.4.0
+  - global
+  - JRA55
   - ryf
```

Comment:
This is my fault, we allowed two options for the forcing mode keywords because I didn't want to be too prescriptive, but it just means we end up with inconsistency. 

My feeling is we should use something that is more descriptive, i.e.`repeatyear`. Also the acronyms are part of the experiment name in most cases, so there is the option to filter on that if users need to find matching experiments.

The QA checks enforce mutual exclusivity, so only one tag from a group is allowed. This is so we don't have `repeatyear` and `interannual` both defined. The downside is we can't have `repeatyear` and `ryf`. 

So .. please change `ryf` to `repeatyear` and sorry!

Reply:
No problem at all! Have changed now :) [7afc228](https://github.com/ACCESS-NRI/access-om2-configs/pull/63/commits/7afc228209040f0227580a45c3d0a245418bb8f1)